docs/BUILD.bazel

```diff
@@ -65,7 +65,7 @@ _READMES = {
     "Jasmine": "//packages/jasmine:README.md",
     "Providers": "//docs:providers.md_",
     "Rollup": "//packages/rollup:README.md",
-    "Terser": "//packages/terser:README.md",
+    "TypeScript": "//packages/typescript:README.md",
```

Comment:
that's strange?

Reply:
Opps, probably a merge conflict 🤔

Reply:
It got deleted in the previous commit, I assume from a (badly resolved) conflict when deleting the `Toolchains` line.